vagrant/provisioning/arkcase-ce-facts.yml

```diff
@@ -376,6 +376,9 @@ default_database_password: "@rKc@3S!"
 tomcat_major_version: 9
 tomcat_version: 9.0.22
 
+# Set Alfresco Admin password
+alfresco_admin_password: "{{ arkcase_admin_password }}" 
+
```

Comment:
Add this to arkcase-dev-facts.yml

Reply:
@jovestefanovski , change this variable to "admin" since the default pass is admin on dev environments